airbyte-integrations/connectors/source-mailchimp/source_mailchimp/manifest.yaml

```diff
@@ -0,0 +1,435 @@
+version: 0.51.2
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - lists
+
+streams:
+  - type: DeclarativeStream
+    name: lists
+    primary_key:
+      - id
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://{{ config.password.split('-')[-1] }}.api.mailchimp.com/3.0/
+        path: lists
+        http_method: GET
+        request_parameters:
+          sort_dir: ASC
+          sort_field: date_created
+        request_headers: {}
+        authenticator:
+          type: BasicHttpAuthenticator
+          password: '{{ config[''password''] }}'
+          username: '{{ config[''username''] }}'
```

Comment:
Connector has two type of auth and the implementation is only using one. Please add the other.